<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicTacToe</class>
 <widget class="QWidget" name="TicTacToe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tic-Tac-Toe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/tacLogo.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/tacLogo.png</iconset>
  </property>
  <property name="toolTip">
   <string extracomment="This is quit button"/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QCommandLinkButton" name="pfButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>109</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>play game with another person</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: aqua
}</string>
   </property>
   <property name="text">
    <string>Play with Friend</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/dir.gif</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/dir.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="pcButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>play game with computer</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: aqua
}</string>
   </property>
   <property name="text">
    <string>Play with Square!</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/dir.gif</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/dir.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="animationLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>620</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;center&gt;&lt;img width=&quot;500&quot; height=&quot;40&quot; src=&quot;:/media/rd_square/Important/Qt projects/TicTacToe/Images/animation.gif&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>419</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Quit from game</string>
   </property>
   <property name="statusTip">
    <string>Quit from game</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: black;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 25;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 25;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Quit Game</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/quit.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/quit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>100</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </widget>
  <widget class="QLabel" name="prodcionLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>310</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: red
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>RD_Square Games presents Tic Tac Toe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>set animation, languate and buttons</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: black;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/settings.gif</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/settings.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="atButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>about game and help</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: black;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>About Game</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/tacLogo.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/tacLogo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="adButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>About game developers</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: black;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 20;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>About Us</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/about.gif</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/about.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </widget>
  <widget class="QLabel" name="wLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Serif</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: aqua
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;body &gt;&lt;p&gt;&lt;center&gt;&lt;font color=&quot;#ff0033&quot;  size=&quot;6&quot;&gt;Welcome to the TIC-TAC-TOE&lt;/font&gt;&lt;/center&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="p1Edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>133</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
color: blue
}</string>
   </property>
   <property name="text">
    <string>Player_1</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="p2Edit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>173</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
color: blue
}</string>
   </property>
   <property name="text">
    <string>Player_2</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="play1Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>133</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: yellow
}</string>
   </property>
   <property name="text">
    <string>Player 1:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="buddy">
    <cstring>p1Edit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="play2Label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>173</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: yellow
}</string>
   </property>
   <property name="text">
    <string>Player 2:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="buddy">
    <cstring>p2Edit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="enterLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>97</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: aqua
}</string>
   </property>
   <property name="text">
    <string>Enter Name of Players</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>213</y>
     <width>101</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>play game</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: red;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 15;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 15;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 4;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Play!</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/play.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </widget>
  <widget class="QLabel" name="winLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Purisa</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: green
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Winner is </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>move to main-menu</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton {
color: red;
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #00ff00;
border-style: solid;
border-radius: 15;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:hover {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #4dff4d, stop: 0.5 #4dff4d, stop: 1 #00b300);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 15;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}

QCommandLinkButton:pressed {
background-color:  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #80ff80, stop: 0.1 #66ff66, stop: 0.49 #80ff80, stop: 0.5 #66ff66, stop: 1 #4dff4d);
border-width: 1px;
border-color: #66ff66;
border-style: solid;
border-radius: 4;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/back.png</normaloff>:/media/rd_square/Important/Qt projects/TicTacToe/Images/back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>settingButton</tabstop>
  <tabstop>pfButton</tabstop>
  <tabstop>pcButton</tabstop>
  <tabstop>atButton</tabstop>
  <tabstop>adButton</tabstop>
  <tabstop>quitButton</tabstop>
  <tabstop>p1Edit</tabstop>
  <tabstop>p2Edit</tabstop>
  <tabstop>playButton</tabstop>
  <tabstop>backButton</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
